driver/postgres/driver.go

```diff
@@ -47,6 +47,10 @@ func (db *postgres) SelectAllMigrations(ctx context.Context) (version.Versions,
 	rows, err := db.connection.QueryContext(ctx, `
 		SELECT version FROM schema_migrations
 	`)
+	if rows.Err() != nil {
```

Comment:
Why this is needed? We already check error below with `if err != nil`

Reply:
There was issue from 'rowserrcheck linter'. Thats why I added this.
```
driver/postgres/driver.go:47:41: rows.Err must be checked (rowserrcheck)
	rows, err := db.connection.QueryContext(ctx, `
```

Reply:
I think this `rows.Err` check should be moved after `rows.Next()` loop, to check if any errors occured during this iteration, according to this http://go-database-sql.org/errors.html

Reply:
> I think this `rows.Err` check should be moved after `rows.Next()` loop, to check if any errors occured during this iteration, according to this http://go-database-sql.org/errors.html

well, ok